thliebig/openEMS-Project
# Fix Homebrew

Fixes #8 

Update Homebrew dependencies. `brew install` doesn't work with URLs on the command line anymore, so `openEMS.rb` and its associated README has been moved to the root directory.

To test:
```shell
brew tap aWZHY0yQH81uOYvH/openems https://github.com/aWZHY0yQH81uOYvH/openEMS-Project.git
git -C /usr/local/Homebrew/Library/Taps/awzhy0yqh81uoyvh/homebrew-openems checkout fix-brew
brew install openems
```

`BREW-README.md` has been modified to match what it should look like in the real repository. I also had to change the Octave path adding instructions since `addpath` resolves symlinks and the true directory pointed to changes based on each update (has commit hash it it).

Also note that the first steps tutorial link is dead (permanently?) — should I change it to something else?

At least one other person should test this out on a clean Mac, and possibly one with an ARM processor.
